### PR TITLE
2.0.0 beta release

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: docs/releases/2.0.0-beta1.rst [hunk 1]
```diff
@@ -1,31 +1,34 @@
 Translate Toolkit 1.14.0
```
Comment (dwaynebailey) [inline docs/releases/2.0.0-beta1.rst]: Version needs changing

Comment (unho) [inline docs/releases/2.0.0-beta1.rst]: Changing it.

File: docs/releases/2.0.0-beta1.rst [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
 
-
-1.14.0 vs 1.14.0-rc1
-====================
+2.0.0-beta1 vs 1.14.0-rc1
+=========================
 
 - Dropped support for Python 2.6.
 - Dropped ``translate.misc.dictutils.ordereddict``.
+- Added support for Python 3.5.
```
Comment (dwaynebailey) [inline docs/releases/2.0.0-beta1.rst]: Expaned Python 3 support to include Python 3.5

File: docs/releases/2.0.0-beta1.rst [hunk 3]
```diff
@@ -46,7 +49,7 @@ Python 3 support
 - Translate Toolkit went through a massive code cleanup looking forward Python
   3 compatibility. There might be quirks that need to be fixed, so please test
   and `report any issue <https://github.com/translate/translate/issues/new>`_
-  you might find.
+  you might find. Python 3.5 is now supported.
```
Comment (dwaynebailey) [inline docs/releases/2.0.0-beta1.rst]: That addition doesn't make any sense.

Just add a bullet pint: Python 3.3-3.5 is now supported

Comment (dwaynebailey) [inline docs/releases/2.0.0-beta1.rst]: Can we specifically credit Claude for the Python 3 support within the major changes listed above.

**Comment by unho** [comment]
Addressed the raised concerns.

**Comment by claudep** [comment]
Thanks for the credits :-)

File: translate/__version__.py [hunk 4]
```diff
@@ -19,16 +19,16 @@
...
 code touching stats or quality checks.  An increased build number will force a
 toolkit user, like Pootle, to regenerate it's stored stats and check
 results."""
 
-sver = "1.14.0-rc1"
+sver = "2.0.0-beta1"
 """Human readable version number. Used for version number display."""
 
-ver = (1, 14, 0)
+ver = (2, 0, 0)
```
Comment (dwaynebailey) [inline translate/__version__.py]: This does remind me that we really should duplicate the versioning stuff from Pootle into ttk.  Just not now.

File: translate/__version__.py [hunk 5]
```diff
@@ -19,16 +19,16 @@
...
-build = 14001
+build = 20000
 """The build number is used by external users of the Translate Toolkit to
 trigger refreshes.  Thus increase the build number whenever changes are made to
 code touching stats or quality checks.  An increased build number will force a
 toolkit user, like Pootle, to regenerate it's stored stats and check
 results."""
 
-sver = "1.14.0-rc1"
+sver = "2.0.0-beta1"
```
Comment (dwaynebailey) [inline translate/__version__.py]: i wonder if we should rather follow semantic versioning where this would be 2.0.0b1 IIRC

Comment (unho) [inline translate/__version__.py]: Will change tomorrow.

File: translate/__version__.py (hunk 4 above)
Comment (unho) [inline translate/__version__.py]: Sure.

File: docs/releases/2.0.0-beta1.rst [hunk 6]
```diff
@@ -47,6 +50,7 @@ Python 3 support
   3 compatibility. There might be quirks that need to be fixed, so please test
   and `report any issue <https://github.com/translate/translate/issues/new>`_
   you might find.
+- Python 3.3-3.5 is now supported. Thanks to Claude Paroz.
```
Comment (dwaynebailey) [inline docs/releases/2.0.0-beta1.rst]: Drop the Claude bit here, above was enough

**Comment by unho** [comment]
Updated. Will land and release this morning.

**Comment by unho** [comment]
It seems @ta2-1 found some issue on Pootle that might stem to TTK, so holding release until he works out the issue.

**Comment by unho** [comment]
The issue seems to be Pootle only, so releasing.

**Pull request merged**